tests/components/home_connect/test_api.py

```diff
@@ -0,0 +1,130 @@
+"""Test the API driver code."""
```

Comment:
These tests belong in a module named `test_init.py`.

Reply:
I can move them over, the only reason I separated the tests was to follow the filename-to-testfile mapping, and the integration has its own API client abstraction in `api.py` as well as the setup in `__init__.py`.

Reply:
I tried moving the tests over to `test_init.py` and it ended up affecting coverage for the `homeassistant/components/home_connect/api.py` file. I decided to leave it in the last commit for now, though can change it easily.

Reply:
Why is coverage affected by moving tests to another module?

Reply:
Please disregard my previous comment. There must have been some issue with my local setup since now when I move over the test functions coverage does not change 🤦